### PR TITLE
refactor: fix clippy lint

### PR DESCRIPTION
**Pull request by kazuk**
## Issue number and link

CI clippy lint failing.

## Describe your changes

fix lint error

## Checklist before requesting a review

- [x] I follow the [Semantic Pull Requests](https://github.com/zeke/semantic-pull-requests) rules (bugfix/feature)
- [ ] I specified links to related issues (must: bugfix, want: feature)
- [ ] I have performed a self-review of my code (bugfix/feature)
- [ ] I have added thorough tests (bugfix/feature)
- [ ] I have edited `## [Unreleased]` section in `CHANGELOG.md` following [keep a changelog](https://keepachangelog.com/en/1.0.0/) syntax (bugfix/feature)
- [ ] I {made/will make} a related pull request for [documentation repo](https://github.com/SpringQL/SpringQL.github.io) (feature)

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -26,6 +26,7 @@ jobs:
         uses: actions/checkout@v2
         with:
           ref: main
+          token: ${{ secrets.GHPAT_FOR_PUSH_RELEASE }}
```
Comment (laysakura) [inline .github/workflows/release.yml:29]: Could you create another PR like `ci: xxx` for this change?

**Review by laysakura — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yml (hunk 1 above)
Comment (kazuk) [reply .github/workflows/release.yml:29]: oh sorry. 
this is unrelated change.

**Review by laysakura — APPROVED** [review]
🙏

**Comment by laysakura** [comment]
Issued CI failure on 1.57.0: https://github.com/SpringQL/SpringQL/issues/240

**Pull request merged**